packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -252,6 +258,10 @@ export default function DataviewsPatterns() {
 	const {
 		params: { postType, categoryId: categoryIdFromURL },
 	} = useLocation();
+	const [ selection, setSelection ] = useState( [] );
+	const onChangeSelection = useCallback( ( items ) => {
+		setSelection( items );
+	}, [] );
```

Comment:
It looks like and identity function, so perhaps the `useCallback` isn't needed and `setSelection` can be passed straight in.

Reply:
Oh, yes. I had some other code at first with deps and forgot to clean up. Thanks!